pages/contributors/submission.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <v-layout justify-center column align-content-center>
     <h1>{{ $t('contributors.submission.title') }}</h1>
-    <woudc-blurb :items="blurb1" />
+    <p>{{ $t('contributors.submission.blurb1') }}</p>
     <span class="note">
       <p>
         <b>{{ $t('note') }} (1)</b>
```

Comment:
The `class="note"` was there because there was supposed to be a commit that added some global CSS files. Somehow it got added to the other pull request instead, but I moved it over.

So added classes like that, and like the occasional `class="newlines"`, now do things.